vendor/wp-now/src/download.ts

```diff
@@ -230,7 +230,18 @@ export async function downloadMuPlugins(customMuPluginsPath = '') {
 			'fonts.gstatic.com',
 		);
 		return array_merge( $hosts, $redirect_hosts );
-	} );`
+	} );
+	add_filter( 'http_request_host_is_external', function( $allow, $host, $url ) {
+		$external_hosts = array(
+			'www.youtube.com',
```

Comment:
I refactored the function to handle those cases ( Actually in all of the examples you provided the host would end up being `www.youtube.com`, so it wouldn't be a problem, but we need that change the function after all, as it will easily resolve https://github.com/Automattic/dotcom-forge/issues/6738 )

Reply:
> but we need that change the function after all, as it will easily resolve https://github.com/Automattic/dotcom-forge/issues/6738

Makes sense - I re-tested, and the refactored changes look good to me. Thanks for updating. 👍